include/klee/Internal/Module/KInstruction.h

```diff
@@ -44,7 +44,8 @@ namespace klee {
 
   public:
     virtual ~KInstruction();
-    void printFileLine(llvm::raw_ostream &);
+    void printFileLine(llvm::raw_ostream &) const;
+    std::string printFileLine() const;
```

Comment:
This looks like a separate refactoring.  I'll add an issue, but I won't address it as part of this PR.